etc/openstack_deploy/user_extras_variables.yml

```diff
@@ -78,3 +78,22 @@ maas_filesystem_threshold: 90.0
 # For an AIO it's recommended to set the following to limit the expected RAM
 # usage of elasticsearch.
 # elasticsearch_heap_size_mb: 1024
+
+# Ship all logs to the logstash container as well as the rsyslog_server.
+# Uncomment the following if the setup-logging play is going to be used.
+# logstash_port: 5544
+# rsyslog_client_user_defined_targets:
+#    - name: "rax-logging"
+#      proto: "tcp"
+#      port: "{{ logstash_port }}"
+#      hostname: "groups['rsyslog_all'][0]"
```

Comment:
something appears to be wrong here, this is processed too literally and I'm not entirely sure about the intent here